contracts/JincorTokenICO.sol

```diff
@@ -152,7 +152,15 @@ contract JincorTokenICO is Haltable, PriceReceiver {
   }
 
   function calculateReferralBonus(uint256 tokens) private returns (uint bonus) {
```

Comment:
uint and int are aliases for uint256 and int256, respectively, may be it would be better to use a single style?

Reply:
You are right, I will change it later so code will look more consistent.